cps.py

```diff
@@ -1,5 +1,15 @@
+import sys
+
 import semantic_version as sv
 
+# Compatibility, without depending on `six`.
+if sys.version_info.major >= 3:
+    iteritems = dict.items
+    unicode = str
+    def map(f, xs): return [f(x) for x in xs]
```

Comment:
Why is this needed?

Reply:
Yeah :( In the downstream Drake PRs, Jeremy suggested surrounding `map()` with `list()` to make it more obvious. Let me know if you'd like to do this here, or if it's OK as-is.

Reply:
That's what I ended up doing in #14... I like that it's more obvious, and also allows using that work-around only when needed.